src/ui/server/content-strings/footer.ts

```diff
@@ -34,3 +22,35 @@ export const FOOTER = {
     HREF: 'https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/',
   },
 };
+
+export const QUOTE_FOOTER = {
+  ...sharedFooter,
+  ACCESSIBILITY_STATEMENT: {
+    TEXT: 'Accessibility statement',
+    HREF: ROUTES.ACCESSIBILITY_STATEMENT,
+  },
+  COOKIES: {
+    TEXT: 'Cookies',
+    HREF: ROUTES.COOKIES,
+  },
+  CONTACT: {
+    TEXT: 'Contact',
+    HREF: ROUTES.CONTACT_US,
+  },
+};
+
+export const INSURANCE_FOOTER = {
+  ...sharedFooter,
+  ACCESSIBILITY_STATEMENT: {
+    TEXT: 'Accessibility statement',
+    HREF: ROUTES.INSURANCE.ACCESSIBILITY_STATEMENT,
+  },
+  COOKIES: {
+    TEXT: 'Cookies',
+    HREF: ROUTES.INSURANCE.COOKIES,
+  },
+  CONTACT: {
+    TEXT: 'Contact',
+    HREF: ROUTES.INSURANCE.CONTACT_US,
+  },
```

Comment:
Lots of repetition here we could just be updating the links in quote/insurance footer definitions?

Reply:
imrpoved